pkg/configmap/configmap_test.go

```diff
@@ -181,3 +193,23 @@ func loadfromFile(t *testing.T, path string) *corev1.ConfigMap {
 
 	return bundle
 }
+
+// equalSlices compares two string slices and has tolerate different orderings
+func equalSlices(a, b []string) bool {
```

Comment:
Do we need a custom testing helper function here? Can we piggy back off of `assert.ElementsMatch(...)` instead?

Reply:
Good point! I made the change